internal/ingress/controller/config/config.go

```diff
@@ -779,14 +778,8 @@ func NewDefault() Configuration {
 	return cfg
 }
 
-// BuildLogFormatUpstream format the log_format upstream using
-// proxy_protocol_addr as remote client address if UseProxyProtocol
-// is enabled.
+// BuildLogFormatUpstream format the log_format upstream.
 func (cfg Configuration) BuildLogFormatUpstream() string {
-	if cfg.LogFormatUpstream == logFormatUpstream {
-		return fmt.Sprintf(cfg.LogFormatUpstream, "$remote_addr")
-	}
-
 	return cfg.LogFormatUpstream
 }
```

Comment:
This function seems to be useless now, maybe just delete it?

Reply:
done